ruby/basics/arrays.md

```diff
@@ -1,28 +1,43 @@
+<!--Chris Pine covers:
+* array literals
+* accessing via index
+* #each & #times methods
+* #to_s, #join
+* #push/#pop, #last, #length-->
+
+<!--I add:
+* array literals/initializing
+* negative indices
+* shift/unshift
+* concatenating/substracting
+* set operations?
+* other methods-->
+
 # Arrays
-At the beginning of this section, you learned about creating and manipulating individual instances of number and string objects. In real-world development, where you'll be working with dozens (and even hundreds!) of such objects and variables, working with individual instances will be tedious, if not impossible.
+At the beginning of this section, you learned about creating and manipulating individual numbers and strings and assigning them to variables. In real-world development, where you'll be working with dozens (and even hundreds!) of variables, working with numbers and strings individually will be tedious, if not impossible.
 
-One way Ruby allows you to represent a collection of such objects is with the use of **arrays**. Rather than working on individual objects at a time, an array allows you to create and manipulate an ordered and indexed collection of such objects (known as **elements** within the array) as a list. An array can contain any combination of Ruby objects (including other arrays), though it is advised that you keep similar objects in any one array.
+One way Ruby allows you to represent a collection of data types is with the use of **arrays**. Rather than working on individual variables, numbers, or strings at a time, an array allows you to create and manipulate an ordered and indexed collection of them (known as **elements** within the array). Think of arrays as a list! An array can contain any combination of variables, numbers, strings, or other Ruby objects (including other arrays), though it is advised that you keep similar data types in any one array.
```

Comment:
I tend to think a list is an insufficient metaphor for arrays. A better suggestion might be numbered bins on a shelf.